chapters/install/ide-speedup.md

```diff
@@ -0,0 +1,90 @@
+# Speed-up Jina App development in PyCharm and VSCode
+
+Jina App development can be time consuming if you donot have the IDE setup correctly. This chapter describes how to speed up your developer experience
+in Pycharm and VS Code IDE while using Jina. In particular we would focus on how to enable Intellisense for Jina YAML configs. Intellisense can provide code completion, showing number and arguments list, validating your YAML config.
```

Comment:
I should expand configs to configuration files. I wouldn't trust that people know the abbreviation

Reply:
- `App` -> app- 
`VS Code` -> `VSCode` (there's no real naming convention except "Visual Studio Code", and the highest alternative that comes up from quick search is `VSCode`)
- `Pycharm` -> `PyCharm`